project_script/datatreatment.py

```diff
@@ -19,3 +21,35 @@ def is_bigger_data_frame(data_frame_1, data_frame_2):
 
 def equalize_data_frame_rows(data_frame_1, data_frame_2):
     return data_frame_1.loc[0:len(data_frame_2)-1, :]
+
+
+def smooth(array):
+    temp_list = []
+    for key, values in enumerate(array):
+        temp_list.append((savgol_filter(array[key], 55, 2).tolist()))
+    return temp_list
+
+
+def find_the_peaks(data_as_list):
+    temp_list = []
+    for key, value in enumerate(data_as_list):
+        temp_list.append(data_as_list[key].max())
+    return temp_list
+
+
+def find_peak_index(array, peak_list):
+    temp_list = []
+    for key, values in enumerate(array):
+        temp_list.append(np.where(array[key] == peak_list[key]))
+    return temp_list
+
+
+def find_time_max(array, index_list):
+    temp_list = []
+    for key, value in enumerate(array):
+        temp_list.append(array[key][index_list[key]].tolist())
+    return temp_list
+
+
+def flatten(t):
```

Comment:
What kind of data this function flatten? Maybe the name could be more readable. Ex: `flatten_something`.

What the parameter T means? If it's an abbreviation of something, i can read more easily if isn't abbreviated. But, if it's a own name of something, that's ok.